test/install.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is for installing node problem detector (NPD) on a running node
+# in standalone mode, as a setup for NPD e2e tests.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly BIN_DIR=/home/kubernetes/bin
+readonly CONFIG_DIR=/home/kubernetes/node-problem-detector/config
```

Comment:
Yeah I choose to reuse the `/home/kubernetes` flavor because it's actually the best way of installing something onto COS.

On COS, `/usr` is read-only, so we cannot install to there. `/etc` is very tiny, with only 1 MB of writable space, which is too small for NPD binary. `/tmp` is no-exec and stateless.

The only usable stateful directory, is `/home` and `/var`. I do not want to put any executable to `/var`. So that leaves us only one option, `/home`.

COS's default filesystem layout is really restrictive :)